### PR TITLE
i3 add rspec feature

### PR DESCRIPTION
**Pull request by duleorlovic**
- [#3] rails g rspec:install
- [#3] Add sample unit, request and system spec

### CODE REVIEW

**Comment by duleorlovic** [comment]
When I learn something new, I try to write in README of that topic, for example
https://github.com/trkin/rails_rspec_tips
https://github.com/trkin/rails_minitest

and put the link to the files (in the first line) when I use them, like https://github.com/trkin/rails_minitest/blob/main/test/fixtures/users.yml#L1

**Comment by DeVuDeveloper** [comment]
Ok, I see you made changes. Let me check it. I will pull or at least take a fresh copy of the repo and test all and read all. 
**I hope you will do the same with my new pull request because I see you read here and don't read a pull request detailed summary. You don't like commits and later you ask for the reason for the change.**
**You cry for 2 lines of code and don't see many other lines.**
**I am on practice here and sure I make mistakes, but I want advice too.
Last, if you don't know to be nice and respect people's time, you should tell it and we will stop here.  I can make all what you want but next time it will be for money.**

File: spec/fixtures/users.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-one:
-  email: one@example.com
+user:
+  <<: *DEFAULTS
 
-two:
-  email: two@example.com
+unconfirmed:
+  <<: *DEFAULTS
+  confirmed_at:
```
Comment (DeVuDeveloper) [inline spec/fixtures/users.yml:12]: Good job so far.
Please explain the source for this code.

File: spec/system/sign_up_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+RSpec.describe "Home management", type: :system do
+  before do
+    driven_by(:selenium_chrome_headless) if ENV["CI"].present?
+  end
+  it "home" do
+    visit "/"
+
+    expect(page).to have_text("Login")
+  end
+end
```
Comment (DeVuDeveloper) [inline spec/system/sign_up_spec.rb:16]: Still browser system test, and I suggest if we are doing disabling the browser.
Can you do a request test instead?

2023-04-10 15:04:52 WARN Selenium [DEPRECATION] [:capabilities] The :capabilities parameter for Selenium::WebDriver::Chrome::Driver is deprecated. Use :options argument with an instance of Selenium::WebDriver::Chrome::Driver instead.
.

**Warning of deprecation.
Can be overridden in support folder, writing index file in rspec helper file and changin capybara settings**.

**Review by DeVuDeveloper — CHANGES_REQUESTED** [review]

File: spec/fixtures/users.yml (hunk 1 above)
Comment (duleorlovic) [reply spec/fixtures/users.yml:12]: There is a link to the source at the top of file

https://github.com/trkin/rails_minitest/blob/main/test/fixtures/users.yml

@trkin/internship the idea is to create common knowledge so we can use those examples in various projects and improve them. I'm still thinking how to organise examples to be easy to understand and compatible with different use case.

File: spec/system/sign_up_spec.rb (hunk 2 above)
Comment (duleorlovic) [reply spec/system/sign_up_spec.rb:16]: > Can you do a request test instead? 

I agree to use requests spec for all our ruby code but currently I have not written any controller or similar ruby code.

This was just a placeholder for feature test so we do not use them to much, I suggest only a few feature tests (like sign up)

> Can be overridden in support folder, writing index file in rspec helper file and changin capybara settings.

Please write an issue and PR that solves this warning.

**Review by DeVuDeveloper — APPROVED** [review]
Approved

**Pull request merged**